### PR TITLE
Pylint luke (red #5) 

### PR DESCRIPTION
**Pull request by ericvaandering**
Another set of bad pylint fixes that will be squashed when approved.

### CODE REVIEW

File: src/python/WMCore/WMBS/Oracle/Workflow/RemoveDuplicates.py [hunk 1]
```diff
@@ -2,10 +2,11 @@
 """
 _RemoveDeuplicates_
 
-Oracle implementation of Workflow.RemoveDeuplicates
```
Comment (amaltaro) [inline src/python/WMCore/WMBS/Oracle/Workflow/RemoveDuplicates.py]: Given it was broken, I believe it's not being used by anyone.

File: src/python/WMCore/WMRuntime/Monitors/TestMonitor.py [hunk 2]
```diff
@@ -205,14 +184,14 @@ def periodicUpdate(self):
                 msg += "WARNING: Hard Kill Timeout has Expired:"
                 logging.error(msg)
                 os.kill(stepPID, signal.SIGTERM)
-                killedpid, stat = os.waitpid(pid, os.WNOHANG)
+                killedpid, stat = os.waitpid(stepPID, os.WNOHANG)
```
Comment (amaltaro) [inline src/python/WMCore/WMRuntime/Monitors/TestMonitor.py]: awesome!

**Comment by amaltaro** [comment]
@ticoann I think we can merge it before cutting the final wmagent release candidate.
@ericvaandering can you please squash it?

**Review by amaltaro — APPROVED** [review]

**Comment by ericvaandering** [comment]
Ok, this is squashed as well. @ticoann please merge when Jenkins comes back OK.

**Comment by ericvaandering** [comment]
@ticoann please merge

**Pull request merged**